src/forecasting/patient_sampler.py

```diff
@@ -26,14 +26,18 @@
 
 
 try:
-    with open(os.path.join(DIRNAME, "specialty_info.json")) as json_file:
+    with open(
+        os.path.join(DIRNAME, "../../data/specialty_info.json")
```

Comment:
With this new directory structure, the scripts can't find specialty_info.json, hourly_elective_prob.json or historic_admissions.csv.

Reply:
Hi Matt can you try generating the fake files and moving them into their final location as per https://github.com/nhsx/skunkworks-bed-allocation/blob/9d418cdd280b1ba80e27829a5f19e3e1885798f3/fake_data_generation/README.md?plain=1#L87 ?

Reply:
Also try running the integration test which is now added: https://github.com/nhsx/skunkworks-bed-allocation/blob/amadeus/integration-tests/docs/integration_testing.md

Reply:
Integration tests runs but UI blank. Will update following meeting later